File: TP3_Python/Codingan/KURSI.py
from Experience import Experience

class KURSI(Experience):
    def __init__(self, tipe_kursi="", kelebihan="", experience_feel=""):
        super().__init__(experience_feel)
        self.tipe_kursi = tipe_kursi
        self.kelebihan = kelebihan

    def set_tipe_kursi(self, tipe_kursi):
        self.tipe_kursi = tipe_kursi

    def set_kelebihan(self, kelebihan):
        self.kelebihan = kelebihan

    def get_tipe_kursi(self):
        return self.tipe_kursi

    def get_kelebihan(self):
        return self.kelebihan

File: TP3_CPP/Codingan/KURSI.cpp
#pragma once
#include "Experience.cpp"
#include <iostream>
#include <string>

using namespace std;

class KURSI : public Experience
{
private:
    string TipeKursi;
    string Kelebihan;

public:
    KURSI()
    {
    }

    KURSI(string TipeKursi, string Kelebihan, string ExperienceFeel) : Experience(ExperienceFeel)
    {
        this->TipeKursi = TipeKursi;
        this->Kelebihan = Kelebihan;
    }

    // Setter
    void setTipeKursi(string TipeKursi)
    {
        this->TipeKursi = TipeKursi;
    }

    void setKelebihan(string Kelebihan)
    {
        this->Kelebihan = Kelebihan;
    }

    // Getter
    string getTipeKursi()
    {
        return TipeKursi;
    }

    string getKelebihan()
    {
        return Kelebihan;
    }

    ~KURSI()
    {
    }
};
